stestr/config_file.py

```diff
@@ -50,6 +52,11 @@ def get_run_command(self, options, test_ids=None, regexes=None):
             test_path = options.test_path
         elif self.parser.has_option('DEFAULT', 'test_path'):
             test_path = self.parser.get('DEFAULT', 'test_path')
+        else:
+            print("no test_path can be found in either the command line "
```

Comment:
I would prefer a capital to start the sentence, so s/no/No. But it's not really worth blocking over